jobs/component_jobs.groovy

```diff
@@ -51,6 +51,31 @@ repos.each { Map repo ->
         authenticationToken('5ISZbTayHuC0nHV')
       }
 
+      if (isPR) { // set up GitHubPullRequest build trigger
```

Comment:
this identical code change is also located in the revert PR at https://github.com/deis/jenkins-jobs/pull/42